### PR TITLE
feat(pci-instances): tapc 1732 onboarding page

### PR DESCRIPTION
**Pull request by fredericvilcot**
| Question         | Answer
| ---------------- | ---
| Branch?          | `pci-instances`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-1732
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

This PR embeds the `/onboarding` page to handle user when no instance has been already created.

### CODE REVIEW

File: packages/manager/apps/pci-instances/public/translations/onboarding/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "not_created_message": "Vous n'avez pas encore créé d’instance.",
```
Comment (SimonChaumet) [inline packages/manager/apps/pci-instances/public/translations/onboarding/Messages_fr_FR.json:2]: Is it the apostrophe from windows ?

File: packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx [hunk 2]
```diff
@@ -42,14 +42,16 @@ import { FilterComparator } from '@ovh-ux/manager-core-api';
...
 
 const initialSorting = {
   id: 'name',
   desc: false,
 };
 
 const Instances: FC = () => {
-  const { t } = useTranslation('list');
+  const { t: tList } = useTranslation('list');
+  const { t: tCommon } = useTranslation('common');
```
Comment (SimonChaumet) [inline packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx:54]: Why not use https://react.i18next.com/guides/multiple-translation-files

**Review by SimonChaumet — APPROVED** [review]

File: packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx (hunk 2 above)
Comment (fredericvilcot) [reply packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx:54]: Yep, agree 👍🏻

File: packages/manager/apps/pci-instances/public/translations/onboarding/Messages_fr_FR.json (hunk 1 above)
Comment (fredericvilcot) [reply packages/manager/apps/pci-instances/public/translations/onboarding/Messages_fr_FR.json:2]: Yes! Good catch!

**Review by SimonChaumet — APPROVED** [review]

**Review by Tsiorifamonjena — APPROVED** [review]

**Pull request merged**